tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -1517,4 +1517,30 @@ public function testPrimaryKeyAutoIncrement()
 
         $this->assertGreaterThan($lastUsedIdBeforeDelete, $lastUsedIdAfterDelete);
     }
+
+    /**
+     * @group 2925
+     */
+    public function testAlterTableChangePrimaryKey(): void
+    {
+        $tableFrom = new Table('primary_key_id');
+        $column    = $tableFrom->addColumn('user_id', 'integer');
+        $column->setAutoincrement(true);
+        $tableFrom->setPrimaryKey(['user_id']);
+        $this->_sm->dropAndCreateTable($tableFrom);
+
+        $tableFrom = $this->_sm->listTableDetails('primary_key_id');
```

Comment:
Bug is in diff method, but functional test is better than check just output because will expose issues on new versions of RDBMS without waiting of issue report.